### ISSUE TITLE
No module named 'src.pyforest' (Installation failure)

### ISSUE THREAD

**Issue by brendalf** [8080labs/pyforest#1]
### Environment

* Operating System:
* Python Version: 3.6.7
* Pip version: 19.1.1
* How did you install pyforest: `pip install pyforest`
* Python packages: 
[req.txt](https://github.com/8080labs/pyforest/files/3498510/req.txt)

### Description of Issue

* What did you expect to happen?
Installation of pyforest

* What happened instead?
```
ERROR: Complete output from command python setup.py egg_info:
ERROR: Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "C:\Users\LUCASM~1\AppData\Local\Temp\pip-install-ibv5qzhr\pyforest\setup.py", line 10, in <module> from src.pyforest.auto_import import setup as setup_auto_import
    ModuleNotFoundError: No module named 'src.pyforest'
```    

### Reproduction Steps

1. `pip install pyforest`

### What steps have you taken to resolve this already?

1. Nothing

### Anything else?

1. In my second computer (python 3.7.3 and pip 19.0.3) the install did work.

**Comment by guido611** [comment]
Hi @brenoufes , thanks for your interest in pyforest. I am Guido from 8080 Labs.

In which environment did you try to install pyforest?

**Comment by FlorianWetschoreck** [comment]
We can reproduce this error with Python 2. However, we cannot reproduce it with Python3.
Please make sure that your pip is using Python3 or maybe use pip3?

**Comment by brendalf** [comment]
Hi @guido611 and @FlorianWetschoreck. Windows 10 with pip 19.1.1 and python 3.6.7

**Comment by haugstve** [comment]
I received the same error message trying to install pyforest on my mac three days ago. I googlet the error message, found this tread, and followed it. 

When trying to reproduce in a clean conda environment today I ended up with a successful install. I have tried it in a few of my other environments and it all went smooth. 

Looking at some terminal history and trying to remember back to the first install makes me belive my problem with pyforest originated when i messed with my .condarc file. 

Sorry for not capturing the environment configurations when the error occurred.
